src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2315,7 +2315,8 @@ private BoundExpression BindExplicitNullableCastFromNonNullable(ExpressionSyntax
                 var underlyingExpr = BindCastCore(node, operand, underlyingTargetTypeWithAnnotations, wasCompilerGenerated: false, diagnostics: bag);
                 if (underlyingExpr.HasErrors || bag.HasAnyErrors())
                 {
-                    Error(diagnostics, ErrorCode.ERR_NoExplicitConv, node, operand.Type, targetTypeWithAnnotations.Type);
+                    var errorCode = bag.AsEnumerable().Last().Code;
+                    Error(diagnostics,  (ErrorCode)errorCode, node, operand.Type, targetTypeWithAnnotations.Type);
```

Comment:
>operand.Type [](start = 68, length = 12)

How do you know these fill-ins are appropriate for whatever the last error code happens to be?  Maybe it is a warning about something irrelevant.